Content.Server/Station/Systems/StationJobsSystem.Roundstart.cs

```diff
@@ -289,8 +290,12 @@ void AssignPlayer(NetUserId player, string job, EntityUid station)
 
             var profile = profiles[player];
             if (profile.PreferenceUnavailable != PreferenceUnavailableMode.SpawnAsOverflow)
+            {
+                var userSession = _playerManager.GetSessionByUserId(player);
+                _chatManager.DispatchServerMessage(userSession, Loc.GetString("job-not-available-wait-in-lobby"));
                 continue;
-
+            }
+            
```

Comment:
it is pure luck that this doesn't fail tests.
You should not be relying on player sessions anywhere within these code paths, find a way to do this from gameticker instead.